docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,165 @@
++ import analytics from "@react-native-firebase/analytics";
+
+// update each request to analytics
+- Analytics.logEvent("test_analytics_event", {
+-  additionaParam: "test",
+- });
++ analytics().logEvent("test_analytics_event", {
++  additionaParam: "test",
++ });
+```
+
+`logEvent` is asynchronous, so be sure to use `await` in front of the function call if you are using it in an `async` function.
+
+## Creating a development build
+
+If you have previously used Expo Go for development and testing, at this point, you will need to migrate to a [development build](/development/introduction/#from-expo-go-to-development-builds), as Expo Go does not include the native code used in React Native Firebase.
+
+[Follow these instructions for creating a development build](/development/getting-started/). Once you are finished, you should have an **eas.json** file that includes a build profile that looks like this:
+
+```json eas.json
+{
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    }
+  }
+}
+```
+If you wish to run the [development build on an iOS simulator](/build-reference/simulators/) instead of a device, include the `ios.simulator` in your profile:
+
+```json eas.json
+{
+  "development": {
+      //..
+      "ios": {
+        "simulator": true
+      }
+    },
+}
+```
+`expo-dev-client` should be present in your **package.json**. Once you run [`eas build`](/build/#running-a-build) with your development build [profile](/build/eas-json/#build-profiles) and platform of choice, you will have an app binary file that you can install on your device or simulator.
+
+If you are already using a development build, rebuild your app, as your native code dependencies have changed.
+
+## Test your changes
+
+Test your changes locally on your Development Build by running:
+
+<Terminal cmd={[
+'$ npx expo start --dev-client'
+]} />
+
+Then follow the prompts on the terminal to run it on your simulator, or scan the QR code to run it on your device. Try an action that will register a Firebase Analytics event. Note that it may take a few minutes for events to show up in the Firebase Console.
+
+## Conclusion
+
+Now that you have your own development build, you can use any package from React Native Firebase library in your development and testing workflow. You're only minutes away from adding [any additional native functionality you'd like](/development/getting-started/#customizing-your-runtime).
+
+## Other migration scenarios
+
+### Analytics for Expo for web
+
+React Native Firebase only supports iOS and Android. If you previously used `expo-firebase-analytics` to send events from an Expo for web client, you will need to use the Firebase JavaScript (JS) SDK. In the Firebase Console, go to Project Settings, and then your web app, and copy the code under "SDK setup and configuration." You can switch between React Native Firebase and Firebase JS requests using the `Platform` component:
+
+```js
```

Comment:
I would recommend using platform extensions instead of Platform.OS, so we don't import code that we don't need on web or native